ppml/trusted-big-data-ml/python/docker-gramine/bigdl-gramine/Dockerfile

```diff
@@ -0,0 +1,39 @@
+ARG BIGDL_IMAGE_NAME
```

Comment:
BIGDL_IMAGE_NAME can be specified as bigdl-ppml-trusted-big-data-ml-python-gramine-base

Reply:
refined in build-image.sh